app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -122,7 +127,10 @@ class CommonTriageQuestionsController @Inject() (
                 ),
               { individualUserType =>
                 val answers    = triageAnswersFomState(state)
-                val redirectTo = redirectToCheckYourAnswers(state)
+                val redirectTo =
+                  if (state.fold(_.isFurtherReturn, _.isFurtherReturn).contains(true))
```

Comment:
you'd have to do `isFurtherReturn` on an updated state with the updated answers in it - `isFurtherReturn` depends on the `IndividualUserType` which has just been submitted in this endpoint. `state` won't always contain the answer that was just submitted